internal/common/baseclient/iito.go

```diff
@@ -0,0 +1,58 @@
+package baseclient
+
+import "github.com/SOMAS2020/SOMAS2020/internal/common/shared"
+
+// RequestGift allows clients to signalize that they want a gift
+// This information is fed to OfferGifts of all other clients.
+// COMPULSORY, you need to implement this method
+func (c *BaseClient) RequestGift() (int, error) {
+	return 0, nil
```

Comment:
Maybe make the base impl return an error?

Reply:
This should actually interact with the client gamestate I think and return a number. When thinking about this could probably do without errors as it so simple